Client/Application/AppDelegate.swift

```diff
@@ -202,7 +204,26 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         self.profile = p
         return p
     }
-    
+
+    func setNightModeBrightness(enabled: Bool = false) {
```

Comment:
`enabled` doesn't need a default.

Also, put this as methods in `NightModeHelper`.